### ISSUE TITLE
tldr client does not handle ctrl+c properly

### ISSUE THREAD

**Issue by sushlala** [tldr-pages/tldr-cpp-client#39]
## Reproduction Rate
everytime

## Steps to Reproduce

> ~ $tldr j
> Local data is older than two weeks, use --update to update it.         >>>>>    this line  is in red
> 
> (ctrl-c here)
> ~ $asdf         >>>>> these are also in red! 
> -bash: asdf: command not found    >>>>> these are also in red! 

![screen shot 2018-10-08 at 1 11 12 pm](https://user-images.githubusercontent.com/12701881/46631394-194f5900-cafc-11e8-9e15-52da86c02b98.png)

## Result
bash text should go back to the way it was before

## Expected Result
bash text is still red

## Additional Information

~ $tldr --version
Local data is older than two weeks, use --update to update it.

tldr v1.3.0 (v1.3.0)
Copyright (C) 2016 Arvid Gerstmann
Source available at https://github.com/tldr-pages/tldr-cpp-client
~ $

**Comment by zdroid** [comment]
This is almost certainly an issue with your custom Bash styling. It exits and works normally on various setups that I tried.

If it still bugs while you're using default shell config, then it's the bug with tldr script and you should notify me to reopen this issue.
